debian/confluent-ksql.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Streaming SQL engine for Apache Kafka
+Documentation=http://docs.confluent.io/
+After=network.target confluent-kafka.target confluent-schema-registry.target
+
+[Service]
+Type=simple
+User=cp-ksql
+Group=confluent
+Environment="LOG_DIR=/var/log/confluent/ksql"
```

Comment:
ksql-run-class (or wahtever it is called) needs to be updated to set the ksql.log.dir property based on LOG_DIR, and then change the config/log4j...properties to use ksql.log.dir rather than the hardcoded /tmp/ksql..